docs/prometheus-integration.md

```diff
@@ -191,6 +191,19 @@ internal-networkpolicy processed
 - **antrea_controller_network_policy_sync_duration_milliseconds:** The
 duration of syncing internal-networkpolicy
 
+#### Antrea Proxy Metrics
+
+- **antrea_proxy_sync_proxy_rules_duration_seconds:** SyncProxyRules duration
+in seconds of AntreaProxy
```

Comment:
Probably move "in seconds" to be after "AntreaProxy".

Reply:
Updated.